core/src/main/java/org/elasticsearch/common/logging/LogConfigurator.java

```diff
@@ -167,7 +167,7 @@ public FileVisitResult visitFile(Path file, BasicFileAttributes attrs) throws IO
     static void loadConfig(Path file, Settings.Builder settingsBuilder) {
         try {
             settingsBuilder.loadFromPath(file);
-        } catch (SettingsException | NoClassDefFoundError e) {
+        } catch (IOException | SettingsException | NoClassDefFoundError e) {
```

Comment:
Any idea why the `NoClassDefFoundError` is here? Is this leftover and no longer needed after the log4j-only migration?